### PR TITLE
Eventing Kafka Broker CI for release-next and release-v1.4

### PR DESCRIPTION
**Pull request by pierDipi**
Generated with https://github.com/openshift-knative/hack/pull/2

### CODE REVIEW

**Comment by matzew** [comment]
```
{  failed to generate steps from config: failed to find a cluster pool providing the cluster: map[architecture:amd64 cloud:aws owner:openshift-ci product:ocp version:4.11]}
```

I think I saw this on my 4.11 attempts as well, looks like CFG is different ?

**Comment by matzew** [comment]
```
racter (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
  * could not run steps: step knative-eventing-mtbroker_filter failed: could not create build knative-eventing-mtbroker_filter: Build.build.openshift.io "knative-eventing-mtbroker_filter" is invalid: metadata.name: Invalid value: "knative-eventing-mtbroker_filter": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

```

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-next__46.yaml [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+- dockerfile_path: openshift/ci-operator/knative-images/event_display/Dockerfile
+  to: knative-eventing-event_display
+- dockerfile_path: openshift/ci-operator/knative-images/heartbeats/Dockerfile
+  to: knative-eventing-heartbeats
+- dockerfile_path: openshift/ci-operator/knative-images/heartbeats_receiver/Dockerfile
+  to: knative-eventing-heartbeats_receiver
+- dockerfile_path: openshift/ci-operator/knative-images/migrate/Dockerfile
+  to: knative-eventing-migrate
+- dockerfile_path: openshift/ci-operator/knative-images/mtbroker_filter/Dockerfile
+  to: knative-eventing-mtbroker_filter
```
[the remark this answers is not in the document]
Comment (matzew) [reply ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-next__46.yaml:31]: the `_` is not allowed

**Comment by pierDipi** [comment]
@matzew I think there aren't 4.11 cluster pools for CI, I'll go back at using 4.10 for now

image

**Pull request merged**